### PR TITLE
[wip] Initial release

### LINKED ISSUE #1: Installation Failure
I figure you're still getting this up and working, but I thought I'd try out an install so I could give you some extra data to work with! here are my failure logs:

https://paste.yunohost.org/raw/wadecudode

### PR DESCRIPTION
**Pull request by Tagadda**
- Close #1 
- #58

### CODE REVIEW

**Comment by Tagadda** [comment]
!bullseye-tagci

**Comment by Tagadda** [comment]
!bullseye-tagci

**Comment by Tagadda** [comment]
Alrighty! (on bullseye)
[Test Badge](https://bleu.cant.at/bullseye-ci/job/4)
[](https://bleu.cant.at/bullseye-ci/job/4)

**Comment by Tagadda** [comment]
[Test Badge](https://bleu.cant.at/bullseye-ci/job/2)
[](https://bleu.cant.at/bullseye-ci/job/2)

**Comment by tituspijean** [comment]
!gogogadgetobullci

**Comment by tituspijean** [comment]
!gogogadgetobullci

**Comment by tituspijean** [comment]
!gogogadgetobullci

**Comment by Tagadda** [comment]
!bullseyetestme

**Comment by tituspijean** [comment]
!testme

**Comment by Tagadda** [comment]
I never did actually implemented the end of the install :sweat_smile:

**Comment by ericgaspar** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

**Comment by yalh76** [comment]
https://ci-apps-dev.yunohost.org/ci/job/4946

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

File: conf/.env.example [hunk 1]
```diff
@@ -19,7 +20,7 @@ CHECK_FOR_UPDATES=check_if_updates_are_available
 
 # EMAIL CONFIGURATION
 DEFAULT_FROM_EMAIL=tooljet@__DOMAIN__
-SMTP_DOMAIN=localhost
+SMTP_DOMAIN=__MAIN_DOMAIN__
 SMTP_PORT=25
```
Comment (alexAubin) [inline conf/.env.example:24]: (maybe there are some options to specify the user/password to auth on the SMTP server, cf the user mail feature in 11.2 ... though it requires manifest v2)

File: doc/DISCLAIMER.md [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
+TODO
 
-* Other infos that people should be aware of, such as:
-    * any specific step to perform after installing (such as manually finishing the install, specific admin credentials, ...)
-    * how to configure / administrate the application if it ain't obvious
-    * upgrade process / specificities / things to be aware of ?
-    * security considerations ?
+### Limitations
+
+* Require YunoHost 11.
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:7]: YunoHost 11 is like one year old now ...

File: doc/DISCLAIMER.md [hunk 3]
```diff
@@ -1,15 +1,9 @@
...
-* SSO does *not* work.
+### After installation
 
-* Any known limitations, constrains or stuff not working, such as (but not limited to):
-    * requiring a full dedicated domain ?
-    * architectures not supported ?
-    * not-working single-sign on or LDAP integration ?
-    * the app requires an important amount of RAM / disk / .. to install or to work properly
-    * etc...
+TODO
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:3]: (to be fixed)

**Comment by tituspijean** [comment]
I suggest to merge #68 beforehand. :)

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

**Pull request merged**